### PR TITLE
Use Thermometer({controller: "TMP36"}) instead of generic anaglogRead(...), with explanation and link to documentation

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

File: server/main.js [hunk 1]
```diff
@@ -160,10 +160,18 @@ board.on("ready", function() {
...
-  board.analogRead("A0", function(raw) {
-    var mV = 5 * 1000 * (raw / 1024);
-    var value = (mV / 10) - 50;
-    temperatureCharacteristic.valueChange(value);
+  // Johnny-Five's Thermometer class provides a built-in
+  // controller definition for the TMP36 sensor. The controller
+  // handles computing a celsius (also fahrenheit & kelvin) from
+  // a raw analog input value.
+  // Reference: http://johnny-five.io/api/thermometer/
+  var temp = new five.Thermometer({
```
Comment (kenchris) [inline server/main.js:165]: Did you try this on the Edison, because I had code exactly like this and this.value was reporting undefined for me

Comment (rwaldron) [inline server/main.js:165]: Yep, but I made a silly mistake, which I explained [here](https://github.com/google/WebFundamentals/pull/2765#issuecomment-203481589)

Updates inbound :)

File: server/main.js [hunk 2]
```diff
@@ -157,10 +157,18 @@ board.on("ready", function() {
...
+    controller: "TMP36",
+    pin: "A0",
+  });
+
+  temp.on("change", function() {
+    temperatureCharacteristic.valueChange(this.celsius);
```
Comment (rwaldron) [inline server/main.js:171]: @kenchris ^^ updated :)

File: server/package.json [hunk 3]
```diff
@@ -8,7 +8,8 @@
   },
   "dependencies": {
     "eddystone-beacon": "^1.0.5",
-    "johnny-five": "^0.9.30",
+    "johnny-five": "^0.9.35",
+    "edison-io": "^0.9.4",
```
Comment (rwaldron) [inline server/package.json:12]: Also, caught this

**Comment by kenchris** [comment]
Not sure now I forgot merging this! :-( sorry

**Comment by rwaldron** [comment]
No worries :)

**Pull request merged**